scenes/board.tscn

```diff
@@ -0,0 +1,67 @@
+[gd_scene load_steps=2 format=3 uid="uid://cd4deyvh3xn52"]
+
+[sub_resource type="GDScript" id="GDScript_tuypk"]
+script/source = "extends Node2D
+
+var startX : int = 350
+var startY : int = 200
+var width : int = 360
+var height : int = width
+var cellWidth : int = width / 3
+var lineWidth : int = 4
+const NUMBER_OF_ROWS : int = 3
+const NUMBER_OF_COLS : int = 3
+const NUMBER_OF_CELLS : int = NUMBER_OF_ROWS * NUMBER_OF_COLS
+var cell_scene = preload(\"res://scenes/cell.tscn\")
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	for i in NUMBER_OF_ROWS:
+		for j in NUMBER_OF_COLS:
+			var cell = cell_scene.instantiate()
+			add_child(cell)
+			cell.position.x = startX + (cellWidth / 2) + (cellWidth * (i % 3))
+			cell.position.y = startY + (cellWidth / 2) + (cellWidth * (j % 3))
+
+	pass
+
+func _draw():
+	draw_line(
+		Vector2(startX, startY + (width / 3)),
+		Vector2(startX + width, startY + (width / 3)), 
+		Color.BLACK, lineWidth, 
+		true
+	)
+	draw_line(
+		Vector2(startX, startY + (width * 2 / 3)),
+		Vector2(startX + width, startY + (width * 2 / 3)), 
+		Color.BLACK, lineWidth, 
+		true
+	)
+	draw_line(
+		Vector2(startX + (width / 3), startY),
+		Vector2(startX + (width / 3), startY + width), 
+		Color.BLACK, lineWidth, 
+		true
+	)
+	draw_line(
+		Vector2(startX + (width * 2 / 3), startY),
+		Vector2(startX + (width * 2 / 3), startY + width), 
+		Color.BLACK, lineWidth, 
+		true
+	)
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
```

Comment:
Do you need to define these built in methods, or did you just leave the boilerplate stubs in?